util/environment/environment_test.go

```diff
@@ -3,11 +3,20 @@
 package environment
 
 import (
+	"fmt"
+	"os"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
+// Makes platform complaint list of values
```

Comment:
s/complaint/compliant :)

Reply:
haha, I have a complaint or two about the Windows platform.